Gruntfile.coffee

```diff
@@ -76,17 +77,6 @@ module.exports = (grunt) ->
             'jsdom-assign'
           ]
         src: 'test/*.coffee'
-      coverage:
-        options:
-          reporter: 'spec'
-          require: [
-            'coffee-script/register'
-            'coffee-coverage/register-istanbul'
-            'jsdom-assign'
-            -> require.cache[require.resolve 'jquery'] = {}
-            'backbone.nativeview'
-          ]
-        src: 'test/*.coffee'
```

Comment:
Do we always want to instrument the code? I've done this this way because IIRC this was not always the case pre-jsdom, but maybe this is just fine to always do it.